website/docs/docs/ai/index.md

```diff
@@ -141,10 +141,46 @@ ERROR: Client error: 403 Forbidden AI credits are exhausted.
 The limits and your usage against these limits are not currently visible on the subscription page. This will be added in due course.
 :::
 
+## Role Based Access Control (RBAC)
+
+In addition to the [system preference](#enabling-the-feature) that enables AI within your workspace, PactFlow's RBAC model allows you to manage access to specific AI capabilities for your users. For detailed information, refer to the [permissions documentation](/docs/permissions#ai).
+
+By default, users assigned the [User](/docs/permissions/predefined-roles#user) role have access to all AI permissions ([`ai:*`](/docs/permissions#ai)), unless:
+
+- Your account was created prior to the release of the AI feature (September 2024), or
+- You have customized your role configuration.
+
+**Granting Access:**
+
+To enable AI access for users, ensure that the assigned role includes either the [`ai:*`](/docs/permissions#ai) permission (granting access to all AI features) or the specific permissions required for the desired functionality. 
+
 ## Getting help and providing feedback
 
 For feedback, feature requests, or assistance with the tool, join our [slack channel](https://pact-foundation.slack.com/archives/C07K2FT0XKK) or speak directly with your Account Manager. For general support, please follow the [usual methods](https://support.smartbear.com/pactflow/message/).
 
 ## Tutorials
 
 We've created an [in-browser tutorial](https://docs.pactflow.io/docs/tutorials#getting-started-with-pactflow-ai) to help you get started with PactFlow's test generation feature.
+
+## Troubleshooting
+
+### `403 Forbidden` error
+
+If you encounter the following error:
+
+```
+ERROR: Client error: 403 Forbidden You do not have the necessary permissions to access this resource, Required permissions: ai:generation:request-response
+```
+
+This indicates that the user does not have the required permissions to access the feature.
+
+**Resolution:**
+
+Ensure the necessary AI [permission](/docs/permissions#ai) is assigned to the user's role. In this example, the `ai:generation:request-response permission` is required. For more details, refer to the [permissions](/docs/permissions#ai) documentation.
+
+### "AI Disabled in System Preferences"
```

Comment:
In addition to these errors, I can think of two others:

1. If the plan itself has AI disabled. No tenant (AFAIK) would have this yet, but it will come up.
2. If they have run out of credits (I can check what the error looks like).